### PR TITLE
styler v1.1.0 

### PR DESCRIPTION
**Pull request by lorenzwalthert**
@jonmcalder if you want you can review this. The preview link is not the netlify deployment, but you can find it when the travis build has completed (at the bottom).

### CODE REVIEW

File: content/posts/2018-10-20-styler-1.1.0.Rmd [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+slug: styler-v1.1.0
+comments: true
+---
+
+[styler](https://github.com/r-lib/styler) 1.1.0 is now available on CRAN. This 
+release introduces new features and is fully backward-compatible. It also
+adapts to changes in the R parser committed into R devel ([#419](https://github.com/r-lib/styler/issues/419)). Major changes
+are:
+
+* styler can now style roxygen code examples in the source code of package.
```
Comment (jonmcalder) [inline content/posts/2018-10-20-styler-1.1.0.Rmd]:
```suggestion
* styler can now style roxygen code examples in the source code of a package.
```

File: content/posts/2018-10-20-styler-1.1.0.Rmd [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* In addition to R and Rmd files, styler can now handle Rnw files too. The API 
+  ignores capitalization and dot-prefixing for file types:
+
+```{r, eval = FALSE}
+styler::style_dir(filetype = c("r", "Rmd", ".rnw"))
+```
+* the print method for the output of `style_text()` (`print.vertical()`) now
+  returns syntax-highlighted code by default, controllable via the option 
+  `styler.colored_print.vertical` using the [prettycode](https://CRAN.R-project.org/package=prettycode) package. This only 
+  works when ran in a console though, so you can c/p this snippet into yours
```
Comment (jonmcalder) [inline content/posts/2018-10-20-styler-1.1.0.Rmd]:
```suggestion
  works when run in a console though, so you can c/p this snippet into your R console
```

File: content/posts/2018-10-20-styler-1.1.0.Rmd [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  to see how it works:
+  
+```{r}
+styler::style_text(c("add <- function(...) {", "purrr::reduce(list(...), `+`)", "}"))
+```
+  
+
+
+There were also quite a few minor changes, the most visible ones to the user 
+were a complete re-design of the README and the fact that the cursor position is
```
Comment (jonmcalder) [inline content/posts/2018-10-20-styler-1.1.0.Rmd]:
```suggestion
are a complete re-design of the README and the fact that the cursor position is 
```

**Comment by lorenzwalthert** [comment]
Thanks Jon. Unfortunately, I have to refactor the netlify deployment because they completely overhauled their API -.-

**Pull request merged**